ChegeBryan/politico
# #163683922 Create party endpoint

#### What does this PR do?

Builds out the API endpoint to create a Party

#### Description of Task to be completed?

- [x] Write tests for the endpoint
- [x] Write tests for data validation
- [x] Validate the user inputs

#### How should this be manually tested?
After cloning the repository to your local machine - check
[README.md](README.md) for instructions checkout to `ft-create-party-api-163683922` branch. [Install the requirements](https://github.com/ChegeBryan/politico#installing) and launch the flask server check [here](https://github.com/ChegeBryan/politico#starting-the-server) for instructions. Using [Postman](https://www.getpostman.com/) open create a request with this header
`Key Content-Type: application/json`.
Put this in the `json` body
```
{
	"party_name": "Actionated Party",
	"hq_address": "Mountain View 23"
}
```

#### What are the relevant pivotal tracker stories?
[#163683922](https://www.pivotaltracker.com/story/show/#163683922)

#### Postman Documentation.
[Postman Docs](https://documenter.getpostman.com/view/4869820/RztoM8mV)

#### Screenshots.
##### Example request on Postman
![screenshot from 2019-02-06 07-57-32](https://user-images.githubusercontent.com/40359451/52321306-1cdce900-29e5-11e9-8acd-2d6b93ce16a1.png)
